lib/tasks/set_parent_thumbnail.rake

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+namespace :hyku do
+  namespace :update do
+    desc 'Make sure all works have a thumbnail'
+    task set_parent_thumbnail: [:environment] do
+      Account.find_each do |account|
+        begin
+          switch!(account.cname)
+          puts "********************** switched to #{account.cname} **********************"
+          Hyrax.config.curation_concerns.each do |cc|
+            next if cc.count.zero?
+
+            puts "********************** checking #{cc}s **********************"
+            parent_work_ids = []
+
+            cc.find_each do |parent_work|
+              next if parent_work.thumbnail
+
+              parent_work.child_works.each do |child_work|
```

Comment:
do we need to account for order somehow? for LV, we sorted the children by identifier then set its parent's thumbnail to be the first sorted child's thumbnail.